app/models/team.rb

```diff
@@ -0,0 +1,4 @@
+class Team < ApplicationRecord
+    belongs_to :trainers
+  end
+  
```

Comment:
It's your project so you can decide on formatting style but in this I noticed that the indentation is off and there's no new line character at the end (that's what the red 🚫  is). That's inconsistent with the rest of your files.

For Measure we use this formatting: https://proctorujira.atlassian.net/l/cp/UDMBMTRg

Reply:
This is helpful!! I will review and adjust my VS code settings.